src/hooks/__tests__/useChainId.test.ts

```diff
@@ -68,9 +68,10 @@ describe('useChainId hook', () => {
       query: {},
     }))
 
-    renderHook(() => useAppDispatch()(setLastChainId('100')))
-
-    const { result } = renderHook(() => useChainId())
+    const { result } = renderHook(() => {
+      useAppDispatch()(setLastChainId('100'))
+      return useChainId()
+    })
```

Comment:
I don't understand how this was passing before. Every call to `render` or `renderHook` now creates a new store, but they were previously resetting the store.

Any insights on this @mikhailxyz?